From: Clement Lefebvre <clement.lefebvre@linuxmint.com>
Date: Mon, 12 Sep 2016 11:49:08 +0100
Subject: Fixed build

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index a15debc..ef46bc7 100644
--- a/configure.ac
+++ b/configure.ac
@@ -11,7 +11,7 @@ m4_define([muffin_plugin_api_version], [3])
 
 AC_INIT([muffin], [muffin_version],
         [https://github.com/linuxmint/muffin/issues])
-AX_IS_RELEASE([git-directory])
+AX_IS_RELEASE([always])
 
 AC_CONFIG_SRCDIR(src/core/display.c)
 AC_CONFIG_HEADERS(config.h)
